python/lancedb/util.py

```diff
@@ -84,6 +85,18 @@ def fs_from_uri(uri: str) -> Tuple[pa_fs.FileSystem, str]:
     return pa_fs.FileSystem.from_uri(uri)
 
 
+def join_uri(base: Union[str, pathlib.Path], *parts: str) -> str:
+    """
+    Join a URI with multiple parts, handling extra environment variables.
```

Comment:
"extra environment variables"?

Reply:
oops that was generated by copilot and I forgot to change it >_<